packages/amplify-category-analytics/provider-utils/awscloudformation/cloudformation-templates/pinpoint-cloudformation-template.json

```diff
@@ -183,6 +204,7 @@
                                 "               Name: appName",
                                 "           }",
                                 "       };",
+                                "       const pinpoint = new aws.Pinpoint({ apiVersion: '2016-12-01', region: event.ResourceProperties.region });",
```

Comment:
If the user tries to use unknown region, the mapping will fail at the Resource Properties level, because the Cloudformation will try to get the `pinpointRegion` from undefined, so this safeguard mechanism will not be triggered. 
The Cloudformation Mappings does not support default value, which causes this limitation. So, to be safe, the mapping keys in the template contains all the current aws regions other than the two dedicated for government uses. All the 9 regions supported by the cli are in the mapping.